src/Widgets/SpaceWidget/SpaceWidgetComponent.js

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import * as Scrivito from "scrivito";
+
+Scrivito.provideComponent("SpaceWidget", ({ widget }) => {
+  const size = widget.get("size") || "medium";
+  const freeSize = size === "free" ? widget.get("freeSize") : "";
+  return (
+    <hr
+      className={`scrivito-space-widget space-widget ${size}`}
+      style={{ height: `${freeSize}` }}
+    />
+  );
+});
```

Comment:
If we change the data layer as suggested above and change the css classes to `hr.scrivito-space-widget.xs` etc. we can simplify the code as this:

```javascript
import * as React from "react";
import * as Scrivito from "scrivito";

Scrivito.provideComponent("SpaceWidget", ({ widget }) => {
  const size = widget.get("size") || "m";
  return <hr className={`scrivito-space-widget ${size}`} />;
});
```